### PR TITLE
Unblocks things on the walls in medbay and also removes an additional Radiation Collector

### LINKED ISSUE #33909: There are two Radiation Collectors in the same tile at 92,113 on Deltastation
See title.

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 
:cl: Dax Dupont
fix: APCs and other wall mounted objects are no longer blocked by the secure medbay storage parts.
fix: Removed a duplicate radiation collector on Deltastation.
/:cl:

[why]: 
https://github.com/tgstation/tgstation/pull/33686 accidently blocked access to some APCs and consoles.
I've moved them around on maps that were affected. 
hqgclpl 1
xj6mtcq 1
kv57hem 1
Also fixes https://github.com/tgstation/tgstation/issues/33909

### CODE REVIEW

**Comment by YPOQ** [comment]
Is the meta APC wired in?

**Comment by DaxDupont** [comment]
oops fixing that i knew i forgot something

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -57749,6 +57741,24 @@
 	pixel_x = 6;
 	pixel_y = -3
 	},
+/obj/item/device/radio/intercom{
+	broadcasting = 0;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dirty var

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -70958,6 +70958,16 @@
 /obj/machinery/disposal/bin,
 /obj/structure/disposalpipe/trunk,
 /obj/effect/turf_decal/bot,
+/obj/machinery/airalarm{
+	dir = 8;
+	pixel_x = 24
+	},
+/obj/machinery/requests_console{
+	department = "Medbay Storage";
+	departmentType = 0;
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: dirty var

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -57749,6 +57741,24 @@
...
+	frequency = 1485;
+	listening = 1;
+	name = "Station Intercom (Medbay)";
+	pixel_x = 30
+	},
+/turf/open/floor/plasteel/white,
+/area/medical/sleeper)
+"QoZ" = (
+/obj/machinery/requests_console{
+	announcementConsole = 0;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dirty var

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 4]
```diff
@@ -33888,6 +33888,13 @@
 	dir = 4;
 	pixel_x = 11
 	},
+/obj/machinery/requests_console{
+	announcementConsole = 0;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dirty var

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by arsserpentarium** [comment]
Please,don't remove additional collectors. They may be needed for guys,who is trying to build another reactor.

**Comment by DaxDupont** [comment]
@arsserpentarium There were two on the same tile. I did not remove any other but the duplicate that was on the same tile.

**Comment by DaxDupont** [comment]
@ShizCalev review again when you can

**Pull request merged**